sound/soc/sof/ipc.c

```diff
@@ -243,26 +222,16 @@ static int ipc_tx_next_msg(struct snd_sof_ipc *ipc)
 	spin_lock_irq(&sdev->ipc_lock);
 
 	/* send first message in TX list */
```

Comment:
@lyakh remove comment if there is not list

Reply:
done, thanks